README.md

```diff
@@ -137,7 +137,7 @@ The function is passed as the child prop:
 | property          | category    | type                      | description                                                                                      |
 |-------------------|-------------|---------------------------|--------------------------------------------------------------------------------------------------|
 | `on`              | state       | `boolean`                 | The current `on` state of toggle                                                                 |
-| `getTogglerProps` | prop getter | `function(props: object)` | returns the props you should apply to the button element you render. Includes `aria-` attributes |
+| `getTogglerProps` | prop getter | `function(props: object)` | returns the props you should apply to the element you render. Includes `aria-` attributes |
```

Comment:
We'll need to add two items to this table.

Reply:
Wil do. Also, if it's all the same to you, I'm starting to think `getElementTogglerProps` is a better name than `getGenericTogglerProps`.

Reply:
That's fine with me :)